Assembly/Whiskey/Tasks/ValidatePathAttribute.cs

```diff
@@ -7,5 +7,9 @@ public sealed class ValidatePathAttribute : Attribute
         public bool Mandatory { get; set; }
 
         public string PathType { get; set; }
+
+        public bool MustExist { get; set; } = true;

```

Comment:
Rename this to `AllowNonexistent` who's default value is `false`. I don't like seeing `MustExist=$false`. It confuses my brain. I want to a true value that changes the default behavior, e.g. `ValidatePath(AllowNonexistent)`.